<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Select">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>신규 상품 수:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>예상 수집 시간:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Products">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Time">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>650</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Run">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="View">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>350</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Run_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="menscare">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>맨즈케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="skincare">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>스킨케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="makeup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>메이크업</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bodycare">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>바디케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="haircare">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>헤어케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="beauty_tool">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>뷰티툴</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="maskpack">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>마스크/팩</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cleansing">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>클렌징</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="suncare">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>선케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fragrance">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>프래그런스</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>98</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>신규 상품코드 수집</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;신규 상품 개체,  리뷰 수집&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>신규 상품 개체,  리뷰 수집</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All Product</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
